packages/components/pie-switch/README.md

```diff
@@ -10,13 +10,27 @@
 
 ## pie-switch
 
-`pie-switch` is a Web Component built using the Lit library.
+`pie-switch` is a Web Component built using the Lit library. It allows users to switch between two states or options with a single action.
 
-This component can be easily integrated into various frontend frameworks and customized through a set of properties.
+## Installation
 
-> [!IMPORTANT]
```

Comment:
This is the nice formatting bit I mean - it was already referencing the docs site! 😂

Reply:
oh I personally thought that was a bit intense! 😂 I can keep the formatting but lose the 'IMPORTANT'?

Reply:
<img width="1138" alt="Screenshot 2023-12-21 at 13 50 02" src="https://github.com/justeattakeaway/pie/assets/16766185/c4d768bf-911f-4593-8b4c-07f67a05501e">

I thought it was quite pretty? :D

Reply:
it's pretty but I think I prefer the consistency of the documentation subtitle. I shared this template a while back in confluence based on what other npm readmes looked like 😄 https://justeattakeaway.atlassian.net/wiki/spaces/WPG/pages/6478562784/Defining+Engineering+Docs+For+Consumers 

<img width="1226" alt="Screenshot 2023-12-21 at 14 12 11" src="https://github.com/justeattakeaway/pie/assets/49618712/59339375-96d1-4fb8-8608-7075ef2e9af4">

I wonder if we could keep the pretty format but use it for something else 😅

Reply:
I guess the good thing about using a title rather than the `!important` is that it's also linkable (if the README ever grows). Does get rid of the issue we saw on NPM where it doesn't format it too 😄